document_retrieval/documentRetrieval/library/postgresql.py

```diff
@@ -45,16 +43,30 @@ def db_return_docs_metadata(self, metric_fn_output):
             ids.append(tupl[0])
         if ids == []:
             raise Exception('No relavant documents for the given query.')
-        t = tuple(ids)
-        SQL = """SELECT document_id,document_source, date, title, celex_num, fulltextlink FROM documents WHERE document_id IN {}""".format(t)
-        docs_metadata = self.execute(SQL)
+        if len(ids) == 1:
+            SQL = ("SELECT document_id,document_source, date, title, celex_num, fulltextlink "
+            "FROM documents WHERE document_id = %s;") 
+            value = str(ids[0])
+            docs_metadata = self.execute(SQL, (value, ))
+        else:
+            t = tuple(ids)
+            SQL = ("SELECT document_id,document_source, date, title, celex_num, fulltextlink "
```

Comment:
Change the SQL statement to
```python
SQL = (
    "SELECT document_id, document_source, date, title, celex_num, fulltextlink "
    "FROM documents WHERE document_id IN %s;"
)
```

Space between `document_id` and `document_source`